bluebottle/time_based/migrations/0095_auto_20240214_1404.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.20 on 2024-02-14 13:04
+
+from django.db import migrations
+
+
+def enable_deadline_activity(apps, schema_editor):
+    InitiativePlatformSettings = apps.get_model('initiatives', 'InitiativePlatformSettings')
+    settings, _create = InitiativePlatformSettings.objects.get_or_create()
+    settings.activity_types.append('deadlineactivity')
```

Comment:
This should only happen when period activities are enabled on your platform right?

Reply:
I kinda of reckoned that all tenants had this, but I'll make a check just to be sure :ok_hand: